src/opamWixMain.ml

```diff
@@ -348,15 +357,78 @@ let create_bundle cli =
       F.copy ~src ~dst;
       F.basename dst, dst
     in
-    let emb_dirs, emb_files = List.partition (fun (_,path) ->
-      let path = resolve_path_aux env path in
-      Sys.is_directory path) conffile.File.Conf.c_embedded
+    let copy_include path src_dir dst_dir =
+      let sep = if Sys.cygwin then '/' else '\\' in
+      let dirs = OpamStd.String.split path sep in
+      let rec aux src dst files =
+        match files with
+        | [] -> ()
+        | [ file ] when Sys.is_directory @@
+          Filename.concat (OpamFilename.Dir.to_string src) file ->
+            let src = OpamFilename.Op.(src/file) in
+            let dst = OpamFilename.Op.(dst/file) in
+            OpamFilename.copy_dir ~src ~dst;
+        | [ file ] ->
+          let src' = OpamFilename.Op.(src//file) in
+          OpamFilename.copy_in src' dst;
+        | file :: files ->
+          let src =  OpamFilename.Op.(src/file) in
+          let dst = OpamFilename.Op.(dst/file) in
+          if not @@ OpamFilename.exists_dir dst then OpamFilename.mkdir dst;
+          aux src dst files
+      in
+      aux src_dir dst_dir dirs
     in
-    let embedded_dirs = List.map (fun (name, dirname) ->
-      copy_embedded (module Dir_impl) dirname name) emb_dirs
+    let emb_modes = List.filter_map (fun (path,alias) ->
+        let path = OpamFilter.expand_string env path in
+        let prefix = OpamPath.Switch.root gt.root st.switch |> OpamFilename.Dir.to_string in
+        match alias with
+        | Some alias -> Some (Copy_alias (path, alias))
+        | _ when OpamStd.String.starts_with ~prefix path ->
+          if not @@ Sys.file_exists path then
+            OpamConsole.error_and_exit `Not_found "Couldn't find embedded %s \
+            in switch prefix." (OpamConsole.colorise `bold path);
+          let path =
+            OpamStd.String.remove_prefix
+              ~prefix:Filename.dir_sep @@
+                OpamStd.String.remove_prefix ~prefix path
+          in
+          begin
+            match String.trim path with
+            | "" ->
+              OpamConsole.warning "Specify a subdirectory of opam-prefix to \
+              include in your installtion. Skipping...";
+              None
+            | _ -> Some (Copy_opam path)
+          end
+        | _ when not (Filename.is_relative path || Filename.is_implicit path) ->
```

Comment:
```suggestion
        | _ when not (Filename.is_relative path) || Filename.is_implicit path ->
```
For external paths, we take only relative paths from current directory, that doesn't contain implicit subpaths.